src/main/java/no/nav/familie/integrasjoner/client/rest/PdlRestClient.kt

```diff
@@ -0,0 +1,71 @@
+package no.nav.familie.integrasjoner.client.rest
+
+import no.nav.familie.http.client.AbstractRestClient
+import no.nav.familie.http.sts.StsRestClient
+import no.nav.familie.http.util.UriUtil
+import no.nav.familie.integrasjoner.felles.OppslagException
+import no.nav.familie.integrasjoner.personopplysning.internal.PdlFødselsDato
+import no.nav.familie.integrasjoner.personopplysning.internal.PdlFødselsdatoRequest
+import no.nav.familie.integrasjoner.personopplysning.internal.PdlFødselsdatoResponse
+import no.nav.familie.integrasjoner.personopplysning.internal.PdlRequestVariable
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Service
+import org.springframework.web.client.RestOperations
+import java.net.URI
+
+@Service
+class PdlRestClient(@Value("\${PDL_URL}") pdlBaseUrl: URI,
+                    @Qualifier("sts") val restTemplate: RestOperations,
+                    private val stsRestClient: StsRestClient)
+    : AbstractRestClient(restTemplate, "pdl.personinfo") {
+
+    private val pdlUri = UriUtil.uri(pdlBaseUrl, PATH_GRAPHQL)
+
+    fun hentFødselsdato(personIdent: String, tema: String): PdlFødselsDato {
+        val pdlFødselsdatoRequest = PdlFødselsdatoRequest(PdlRequestVariable(personIdent))
+        try {
+            val response = postForEntity<PdlFødselsdatoResponse>(pdlUri,
+                                                                 pdlFødselsdatoRequest,
+                                                                 httpHeaders())
+
+            if (response != null && !response.harFeil()) {
+                return  response?.data?.person?.foedsel?.get(0) ?:
+                        throw OppslagException("Fant ikke forespurte data på person $personIdent",
+                                               "PdlRestClient",
+                                               OppslagException.Level.LAV,
+                                               HttpStatus.NOT_FOUND)
+
+            } else {
+                responsFailure.increment()
+                throw OppslagException("Feil ved oppslag på personinfo ${response?.errors?.toString()}",
```

Comment:
Kan det være personinfo i feilen her? Isåfall bør det kun logges til securelogger. Jeg _tror_ det logges til stdout ved LAV level.

Reply:
Det hadde du helt rett i, så det må vi unngå. Good catch!